src/vs/platform/environment/node/argv.ts

```diff
@@ -62,8 +62,8 @@ export interface ParsedArgs {
 	'export-default-configuration'?: string;
 	'install-source'?: string;
 	'disable-updates'?: boolean;
-	'disable-crash-reporter'?: boolean;
```

Comment:
@deepak1556 should we keep `disable-crash-reporter` in `argv.ts` just because we read it from `main.js` still? I think it helps document the list of CLI flags we support, even if only used in one place.